module/VuFind/src/VuFind/Search/Solr/Results.php

```diff
@@ -346,9 +349,21 @@ public function getFacetList($filter = null)
                     $displayText = $this->hierarchicalFacetHelper
                         ->formatDisplayText($displayText);
                 }
-                $currentSettings['displayText'] = $translate
-                    ? $this->translate([$translateTextDomain, $displayText])
-                    : $displayText;
+                $currentSettings['displayText'] = $displayText;
```

Comment:
I think this would be better as an `else` branch to the `if ($translate)` branch below.